addon_service/oauth1/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+from addon_service.common.base_model import AddonsServiceBaseModel
+
+
+class OAuth1ClientConfig(AddonsServiceBaseModel):
+    """
+    Model for storing attributes that are required for managing
+    OAuth1 credentials exchanges with an ExternalService on behalf
+    of a registered client (e.g. the OSF)
+    """
+
+    # URI that allows to obtain temporary request token to proceed with user auth
+    request_token_url = models.URLField(null=False)
+    # URI to which user will be redirected to authenticate
+    auth_url = models.URLField(null=False)
```

Comment:
I dunno, If you say so, maybe it is

Reply:
In this case, it' guaranteed to be a url. I think the only issue right now is that we don't really have a standard for when we use `url` vs `uri`, which is something we should settle on as part of code cleanup and documentation tasks.

Reply:
at risk of hyper-bikeshedding... i'm gently in favor of `url` when it will be used to make requests and must resolve or something's wrong (as for `auth_url`) -- in contrast to `user_uri`/`resource_uri` that we use as identifiers but don't send requests to directly

Reply:
(recording result of previous discussion: we want to use uri/iri for things that are purely references, and url for things that will be used for requests, @aaxelb can own this as part of code cleanup tasks)